packages/components/Avatar/src/AvatarTokens.ts

```diff
@@ -198,4 +198,4 @@ function getColorProps(color: string, theme: Theme) {
         ringColor: theme.colors.transparentStroke,
       };
   }
-}
+}
```

Comment:
nit: this should be reverted. I think you'll want to run prettier across the files.